setup.py

```diff
@@ -208,6 +213,8 @@ def members_appended(*ds):
         ext_modules.append(freebsd_ext)
     elif sys.platform == 'darwin':
         ext_modules.append(darwin_ext)
+    elif sys.platform == 'win32':
+        ext_modules.append(windows_ext)
```

Comment:
maybe move upwards, see how it checks for not-win32 some lines above already.

could also use is_win32 there....